venues/NeurIPS.cc/2021/Conference/python/build-conflicts.py

```diff
@@ -0,0 +1,227 @@
+import openreview
+import argparse
+
+
+def get_profile_info(i):
+
+    if '@' in i:
+        profile=client.search_profiles(emails=[i])[i]
+    elif '~' in i:
+        profile=client.search_profiles(ids=[i])[0]  
+    try:
+        test=profile.content
+    except:
+        return "profile not found: {}".format(i)
+    
+    domains = set()
+    emails=set()
+    relations = set()
+    common_domains = ['gmail.com', 'qq.com', '126.com', '163.com', 
+                      'outloook.com', 'hotmail.com', 'yahoo.com', 'aol.com', 'msn.com', 'ymail.com', 'googlemail.com', 'live.com']
+
+    ## Institution section, get history within the last three years
+    for h in profile.content.get('history', []):
+        if h.get('end') is None or int(h.get('end')) > 2017:
+            domain = h.get('institution', {}).get('domain', '')
+            domains.update(openreview.tools.subdomains(domain))
+
+    ## Relations section, get coauthor/coworker relations within the last three years + all the other relations
+    for r in profile.content.get('relations', []):
+        if r.get('relation', '') in ['Coauthor','Coworker']:
```

Comment:
You may want to double-check this with Alina, but I think she wants to get the co-authors relations from the publications too. A way to get the publications for a specific profile:

```
openreview.tools.iterget_notes(client, content={'authorids': profile_id})
```

I think the way to check this is: if there is an intersection of publications between an author and a reviewer then there is a conflict.

Reply:
You may also want to use the publications from the last 3 years only.

Reply:
I added this part to the script. 
On the dev site: openreview.tools.iterget_notes(client, content={'authorids': '~Alina_Beygelzimer1'}) also shows the test submission I created. 
Is it true that the results will include the user's current submissions at neurips2021? 
Will the results include the user's submissions at other conferences on openreview?

Reply:
Yes, that is correct and they should be to detect the conflicts. Do you see a problem with that?